content/en/observability_pipelines/destinations/google_cloud_storage.md

```diff
@@ -23,7 +23,22 @@ You need to have Datadog's [Google Cloud Platform integration][3] installed to s
 
 Set up the Google Cloud Storage destination and its environment variables when you [set up an Archive Logs pipeline][4]. The information below is configured in the pipelines UI.
```

Comment:
I recognize this is not part of your update, but just flagging that this link is currently giving a `404` error. I would have just opened a separate PR to fix it myself, but after a quick search of the docs, I'm not sure what the expected destination is.

Reply:
Thanks for this!! This actually redirects in prod because it's a redirect that docs-eng had setup. However, it shouldn't be relying on the redirect so I updated the link.